api/src/controllers/robiot.controller.ts

```diff
@@ -1,15 +1,20 @@
-import { Request, Response } from 'express';
+import { Request, Response, NextFunction } from 'express';
+import { body, validationResult } from 'express-validator';
 import '../config/passport';
-import { getRobiotBattery, getRobiotPosition, getRobiotStatus, moveRobiot } from '../services/robiot.service';
 import logger from '../util/logger';
+import asyncHandler from '../util/asyncHandler';
+import { checkUniqueDirection, checkFloatFormat } from '../validators/robiot.validator';
+
+import { getRobiotBattery, getRobiotPosition, moveRobiot, getRobiotStatus } from '../services/robiot.service';
 
 /**
  * GET /api/v1/robiot/
  * get Config list from one ROBIOT
  */
-export const getConfigListFromRobiot = async (req: Request, res: Response): Promise<void> => {
-    const result = await getRobiotPosition();
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const getConfigListFromRobiot = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Codacy found an issue: ['next' is defined but never used.](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941354)